velero-2-mixin/panels.libsonnet

```diff
@@ -90,5 +90,175 @@ local utils = commonlib.utils;
           targets=[t.topClustersByCSISnapshotSuccess, t.topClustersByCSISnapshotFailure],
           description='Top clusters by number of CSI snapshots.'
         ),
+
+      backupSuccessRate:
+        g.panel.gauge.new('Backup success rate (1 hour)')
+        + g.panel.gauge.queryOptions.withTargets([t.backupSuccessRate])
+        + g.panel.gauge.queryOptions.withDatasource('prometheus', '${prometheus_datasource}')
+        + g.panel.gauge.panelOptions.withDescription('Success rate of backups.')
+        + g.panel.gauge.standardOptions.withUnit('percentunit'),
+
+      restoreSuccessRate:
+        g.panel.gauge.new('Restore success rate (1 hour)')
+        + g.panel.gauge.queryOptions.withTargets([t.restoreSuccessRate])
+        + g.panel.gauge.queryOptions.withDatasource('prometheus', '${prometheus_datasource}')
+        + g.panel.gauge.panelOptions.withDescription('Success rate of restores.')
+        + g.panel.gauge.standardOptions.withUnit('percentunit'),
+
+      volumeSnapshotSuccessRate:
+        g.panel.gauge.new('Volume snapshot success rate')
+        + g.panel.gauge.queryOptions.withTargets([t.volumeSnapshotSuccessRate])
+        + g.panel.gauge.queryOptions.withDatasource('prometheus', '${prometheus_datasource}')
+        + g.panel.gauge.panelOptions.withDescription('Success rate of volume snapshots.')
+        + g.panel.gauge.standardOptions.withUnit('percentunit'),
+
+      restoreValidationFailure:
+        commonlib.panels.generic.stat.info.new(
+          'Restore validation failure / $__interval ',
+          targets=[t.restoreValidationFailure],
+          description='Number of failed restore validations'
+        )
+        + stat.options.withGraphMode('area'),
+      lastBackupStatus:
+        commonlib.panels.generic.stat.base.new(
+          'Backup last status',
```

Comment:
I see what you are saying with the implication. but at the same time I can't help but feel this is one of those spots where you want to be purposefully verbose, just to avoid any possible confusion.